src/pages/on-chain-governance/index.page.tsx

```diff
@@ -446,7 +464,8 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       currentBlockCount,
```

Comment:
see if we can remove `JSON.parse(JSON.stringify(..))` at line 462 and 463 now after using WhaleApiClient